src/sentry/integrations/utils/commit_context.py

```diff
@@ -52,6 +52,7 @@ def find_commit_context_for_event(
                 "integrations.failed_to_fetch_commit_context",
                 organization_id=code_mapping.organization_integration.organization_id,
                 project_id=code_mapping.project.id,
+                group_id=extra["group"],
```

Comment:
We know `extra["group"]` exists because if is defined [here](https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/commit_context.py#L73) and passed into the `find_commit_context_for_event` function [here](https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/commit_context.py#L149)